docs/jwst/package_index.rst

```diff
@@ -21,6 +21,7 @@ Package Index
    extract_2d/index.rst
    fits_generator/index.rst
    flatfield/index.rst
+   firstframe/index.rst
```

Comment:
OK, this is hopelessly picky, but to maintain strict alphabetical listing, firstframe should appear between extract_2d and fits_generator.

Reply:
Done.